packages/pretty-format/src/__tests__/pretty-format-test.js

```diff
@@ -640,9 +640,7 @@ describe('prettyFormat()', () => {
     });
 
     it('supports a single element with custom React elements with props', () => {
-      function Cat() {
-        return React.createElement('div');
-      }
+      const Cat = () => React.createElement('div');
```

Comment:
you'll need to keep using `function` here because Node 4 cannot infer the function name properly.